KenticoOnboardingApplication/KenticoOnboardingApplication.Api.Tests/packages.config

```diff
@@ -0,0 +1,5 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
```

Comment:
Uninstall these two packages and install NUnit (use NugetPackageManager). NUnit is better for continuous integration are is commonly used.

![image](https://user-images.githubusercontent.com/16819859/45361028-6f021580-b5d1-11e8-8de0-588cf8e68cd3.png)